docs/changes/io.fits/11922.bugfix.rst

```diff
@@ -0,0 +1 @@
+Implemented ``np.float32`` instance checking for FITS headers values. A ``ValueError`` should now raise if an ``np.float32`` value is beign assigned for a FITS header.
```

Comment:
Please rewrite the second sentence as:

    A ``ValueError`` is now raised if an ``np.float32`` value is assigned to a header.

Reply:
Should be `an ``np.float32`` NaN/Inf` right ?

Reply:
Right, thank you.